### ISSUE TITLE
Feature request: Adding tokens via image load

### ISSUE THREAD

**Issue by GioDel** [dungeon-revealer/dungeon-revealer#931]
I already like the colored moveable tokens with a text on these. It helps a lot to simulate fights on a map. 
What do you think of the idea to enhance the exiting token feature?

I'm thinking of having a image as token. So instead of having a colored token named "unicorn" I would like to relate an image to it which shows a unicorn. Maybe the token could have a colored border to distinguish between friend and enemy.

Cheers, Giovanni

**Comment by n1ru4l** [comment]
Duplicate of https://github.com/dungeon-revealer/dungeon-revealer/issues/173 🙂

Currently blocked by https://github.com/dungeon-revealer/dungeon-revealer/pull/925 but will be the priority once it is finished 😉

**Comment by n1ru4l** [comment]
implemented on master will be released as part of v1.14.0
